content/sensu-enterprise/2.6/guides/using-contact-routing.md

```diff
@@ -0,0 +1,623 @@
+In the following section we'll be going over different scenarios and showcasing how contact routing would be used and how contact routing affects notifications being sent. We'll be using Email, Slack and PagerDuty as our example integrations/handlers.
+
+Bellow is the default configuration for the integrations we'll be working with in our examples:
+
+{{< highlight json >}}
+{
+  "slack": {
+    "webhook_url": "https://hooks.slack.com/services/foo/bar/foobar",
+    "username": "sensu",
+    "channel": "#alerts",
+    "timeout": 10
+  },
+  "email": {
+    "smtp": {
+      "address": "smtp.example.com",
+      "port": 587,
+      "openssl_verify_mode": "none",
+      "enable_starttls_auto": true,
+      "authentication": "plain",
+      "user_name": "postmaster@example.com",
+      "password": "SECRET"
+    },
+    "to": "default@example.com",
+    "from": "noreply@example.com",
+    "timeout": 10
+  },
+  "pagerduty": {
+    "service_key": "someservicekey ",
+    "timeout": 10
+  }
+}
+{{< /highlight >}}
+
+## Single Handler
+
+### Single Handler with a Single Matching Contact
+
+For our first configuration we have a single handler **email** and a contact **support** configured. The **support** contact has a configuration override for **email** that will change the default _to_ email address from "default@example.com" to "support@sensuapp.com".
+
+{{< highlight json >}}
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handlers": [
+        "email"
+      ],
+      "contacts": [
+        "support"
+      ]
+    }
+  },
+  "contacts": {
+    "support": {
+      "email": {
+        "to": "support@sensuapp.com"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+Because the contact configuration matches the handler being used, any event generated for this check will use this contact's configuration.
+
+![Single Handler with Single Matching Contact](/images/contact-routing/single-handler/single-matching-contact.png)
+
+### Single Handler with a Single Non-Matching Contact
+
+This configuration has a single handler **email** a contact **support** configured. The **support** contact does not have a configuration override for **email**.
+
+{{< highlight json >}}
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handlers": [
+        "email"
+      ],
+      "contacts": [
+        "support"
+      ]
+    }
+  },
+  "contacts": {
+    "support": {
+      "pagerduty": {
+        "service_key": "foobarkey"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+Even though the contact **support** matches in both the **checks** and **contacts**, since there is no configuration under the **support** contact for **email**, no change to the default email configuration is made and the default configuration is used. In our example the email will be sent to "default@example.com".
+
+![Single Handler with Single Non-Matching Contact](/images/contact-routing/single-handler/single-non-matching-contact.png)
```

Comment:
This image shows the event going to `support@sensuapp.com`, rather than what is called out in the scenario above of `default@example.com`. I believe `default@example.com` is the correct on since there is no matching contact override for `email` and `default@example.com` is in the `email` handler.

Reply:
updated image